Misc/NEWS.d/next/Library/2020-04-06-11-05-13.bpo-40196.Jqowse.rst

```diff
@@ -0,0 +1,2 @@
+Fix a bug in the ``symtable`` module that was causing incorrectly report
```

Comment:
nit
```
:mod:`symtable`
```

Reply:
Oh, I was under the impression that both produce the same sphinx output?

Reply:
FYI, it generates different output, The latter creates a hyperlink to the module
```
``symtable``
```
<img width="915" alt="스크린샷 2020-04-07 오전 12 04 52" src="https://user-images.githubusercontent.com/5110323/78573315-8ebbf700-7863-11ea-9f43-bf62c3d4f98e.png">

```
:mod:`symtable`
```

<img width="808" alt="스크린샷 2020-04-07 오전 12 04 10" src="https://user-images.githubusercontent.com/5110323/78573298-895eac80-7863-11ea-82e1-35cf6f76555f.png">

Reply:
So end-user can move to the module more easily :)

Reply:
Wow, I have been living in a lie! Thanks for pointing this out :)